includes/sdk-missing-migration.md

```diff
@@ -0,0 +1,4 @@
+!!! warning "Migration logic not available"
+    The maintenance SDK stores device ID, user ID, etc., in SQLite. However, there is no migration logic available (at this time), so this data won't be automatically migrated when upgrading to the latest SDK. Consequently, for example, device ID may be regenerated when using the latest SDK.
+
```

Comment:
For swift, the un-send event won't be migrated either.  We should also add that.

Reply:
Do we migrate un-send events for Android?

Reply:
It seems we don't. Will have this in Android Migration doc as well.

Reply:
The migration plugin in Kotlin will send any pending events from storage

Reply:
Thanks Justin, I found the PR https://amplitude.atlassian.net/browse/AMP-52578